tests/model_registry/rbac/conftest.py

```diff
@@ -89,6 +92,51 @@ def sa_token(service_account: ServiceAccount) -> str:
         raise
 
 
+@pytest.fixture(scope="function")
+def new_group(
+    request: pytest.FixtureRequest, test_idp_user_session: TestUserSession, admin_client: DynamicClient
+) -> Generator[str, None, None]:
+    """
+    Fixture to create a new OpenShift group and add a user, then delete the group after the test.
+    The parameter should be a tuple: (group_name, user_name).
```

Comment:
It doesn't look like this is true? you get the username from the `test_idp_user_session`